frontend/src/metabase/query_builder/components/view/ViewHeader.jsx

```diff
@@ -260,6 +262,26 @@ export class ViewTitleHeader extends React.Component {
               />
             </Box>
           )}
+          {question.query().database() && isNative && isSaved && (
+            <Link
+              to={new Question(
+                {
+                  dataset_query: {
+                    database: question.query().database().id,
+                    query: { "source-table": `card__${question.id()}` },
+                    type: "query",
+                  },
+                  display: "table",
+                  visualization_settings: {},
+                },
+                question.metadata(),
+              ).getUrl()}
```

Comment:
since `new Question(...).getUrl()` is mostly unrelated to rendering logic can you pull it out into a separate function?